lasagne/layers/cuda_convnet.py

```diff
@@ -71,23 +71,29 @@ class Conv2DCCLayer(CCLayer):
         convolution operation. This layer does not support using different
         strides along both axes.
 
-    border_mode : str, one of 'valid', 'full', 'same'
-        A string indicating the convolution border mode.
+    pad : int, tuple of int, 'full' or 'same' (default: 0)
+        By default, the convolution is only computed where the input and the
+        filter fully overlap (a valid convolution). When ``stride=1``, this
+        yields an output that is smaller than the input by ``filter_size - 1``.
+        The `pad` argument allows to implicitly pad the input with zeros,
+        extending the output size.
 
-        If 'valid', the convolution is only computed where the input and the
-        filter fully overlap.
+        A single integer results in symmetric zero-padding of the given size on
+        all borders, a tuple of two integers allows different symmetric padding
+        per dimension.
```

Comment:
Different padding per dimension is not supported by this layer (this was included in the previous `pad` description), so the description is a bit misleading here. Maybe:
```
A single integer results in symmetric zero-padding of the given size on
all borders. This layer does not support using different amounts of
padding along both axes, but for compatibility to other layers you can
still specify the padding as a tuple of two same-valued integers.